unit_tests/engine/base64.cpp

```diff
@@ -43,7 +43,7 @@ BOOST_AUTO_TEST_CASE(hint_encoding_decoding_roundtrip)
 
     const Coordinate coordinate;
     const PhantomNode phantom;
-    const osrm::test::MockDataFacade<osrm::engine::algorithm::CH> facade{};
+    const osrm::test::MockDataFacade<osrm::engine::datafacade::CH> facade{};
```

Comment:
I think it might be a little bit confusing if we use the `datafacade` internal name here. Doing something like:

`using CH = osrm::engine::ch::algorithm` would be more obvious when reading this code.